<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dizering</class>
 <widget class="QWidget" name="Dizering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="bitnost">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Битность</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="noDizering">
     <property name="text">
      <string>Без дизеринга</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ordDizering">
     <property name="text">
      <string>Ordered dizering(8x8)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="randDizering">
     <property name="text">
      <string>Random dizering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="floydDizering">
     <property name="text">
      <string>Floyd-Steinberg dizering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="atkiDizering">
     <property name="text">
      <string>Atkinson dizering</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="showDiz">
       <property name="text">
        <string>Предпросмотр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveDiz">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
